lib/rules/web/admin_console/base/login.xyaml

```diff
@@ -9,6 +9,7 @@ verify_logged_in_admin_console:
     selector: |-
       var ns = window.localStorage['exp'];
       return ns ? document.documentElement : null;
+    timeout: 120
```

Comment:
Add timeout here since sometimes `window.localStorage` is not written very timely then js will timeout